modules/openapi-generator/src/main/resources/python/python-experimental/signing.mustache

```diff
@@ -0,0 +1,317 @@
+
+HEADER_REQUEST_TARGET = '(request-target)'
+HEADER_CREATED = '(created)'
+HEADER_EXPIRES = '(expires)'
+HEADER_HOST = 'host'
+HEADER_DATE = 'date'
+HEADER_DIGEST = 'Digest'
+HEADER_AUTHORIZATION = 'Authorization'
+
+SCHEME_HS2019 = 'hs2019'
+SCHEME_RSA_SHA256 = 'rsa-sha256'
+SCHEME_RSA_SHA512 = 'rsa-sha512'
+
+ALGORITHM_RSASSA_PSS = 'PSS'
+ALGORITHM_RSASSA_PKCS1v15 = 'PKCS1v15'
+
+ALGORITHM_ECDSA_MODE_FIPS_186_3 = 'fips-186-3'
+ALGORITHM_ECDSA_MODE_DETERMINISTIC_RFC6979 = 'deterministic-rfc6979'
+ALGORITHM_ECDSA_KEY_SIGNING_ALGORITHMS = {
+    ECDSA_MODE_FIPS_186_3,
+    ECDSA_MODE_DETERMINISTIC_RFC6979
+}
+
+class HttpSigningConfiguration(object):
+    """NOTE: This class is auto generated by OpenAPI Generator
+
+    Ref: https://openapi-generator.tech
+    Do not edit the class manually.
+
+    :param key_id: The identifier of the cryptographic key, when signing HTTP requests.
+        An 'Authorization' header is calculated by creating a hash of select headers,
+        and optionally the body of the HTTP request, then signing the hash value using
+        a private key which is available to the client.
+    :param private_key_path: The path of the file containing a private key,
+        when signing HTTP requests.
+    :param signing_scheme: The signature scheme, when signing HTTP requests.
+        Supported value is hs2019.
+    :param signed_headers: A list of strings. Each value is the name of a HTTP header
+        that must be included in the HTTP signature calculation.
+        The two special signature headers '(request-target)' and '(created)' SHOULD be
+        included in SignedHeaders.
+        The '(created)' header expresses when the signature was created.
+        The '(request-target)' header is a concatenation of the lowercased :method, an
+        ASCII space, and the :path pseudo-headers.
+        When signed_headers is not specified, the client defaults to a single value,
+        '(created)', in the list of HTTP headers.
+        When SignedHeaders contains the 'Digest' value, the client performs the
+        following operations:
+        1. Calculate a digest of request body, as specified in RFC3230, section 4.3.2.
+        2. Set the 'Digest' header in the request body.
+        3. Include the 'Digest' header and value in the HTTP signature.
+    :param signing_algorithm: The signature algorithm, when signing HTTP requests.
+        Supported values are PKCS1v15, PSS; fips-186-3, deterministic-rfc6979.
+    :param signature_max_validity: The signature max validity,
+        expressed as a datetime.timedelta value.
+    """
+    def __init__(self, key_id, private_key_path,
+                 signing_scheme=SCHEME_HS2019,
+                 signed_headers=[HEADER_CREATED],
+                 signing_algorithm=None,
+                 signature_max_validity=None):
+        """Constructor
+        """
+        self.key_id = key_id
+        """The identifier of the key used to sign HTTP requests.
+        """
```

Comment:
Can you have all the docstring info on these parameters in the top of the class or here in the `__init__` dosctring but not both?
Python std practice is to put it in one of those places but not both. So if you want to keep it in the class can you delete these comments describing these params?

Reply:
doing...

Reply:
done.